CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
-## 1.2.0-dev
+## 1.2.0-nnbd
 
+* Migrate to NNBD
 * Add typed queue classes such as `Uint8Queue`. These classes implement both
   `Queue` and `List` with a highly-efficient typed-data-backed implementation.
   Their `sublist()` methods also return typed data classes.
-
 * Update min Dart SDK to `2.4.0`.
```

Comment:
This change no longer applies. Should we try to get another publish out first?

Reply:
I would be fine with that

Reply:
done